### PR TITLE
Changed update_entity to update_token

### LINKED ISSUE #114: Change public method name update_entity
Change public method name update_entity to `update_token`

### PR DESCRIPTION
**Pull request by neilong31**
Fixes #114 
Every instance of the method update_entity has been renamed or is calling update_token now.

### CODE REVIEW

File: flask_jwt_router/_jwt_routes.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def register():
         def login():
             # I'm authorized & updating my token!
             return jsonify({
-                "token": jwt_routes.update_entity(entity_id=1)
```
Comment (joegasewicz) [inline flask_jwt_router/_jwt_routes.py]: If you can instal `Sphinx` &  run the following:
```
make html
```
I will update the Pipfile to add Sphinx as a dev dep in my Pr either today or tomorrow, thank you.

Comment (neilong31) [reply flask_jwt_router/_jwt_routes.py]: Okay I'll try and give it a go!

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,4 +4,8 @@
 
 ### Changed
 
--   Replaced the the `entity_type` kwarg to `table_name` in the public method `register_entity` [Issue #111](https://github.com/joegasewicz/Flask-JWT-Router/issues/111)
\ No newline at end of file
+-   Replaced the the `entity_type` kwarg to `table_name` in the public method `register_entity` [Issue #111](https://github.com/joegasewicz/Flask-JWT-Router/issues/111)
+
+-   Renamed the `update_entity` public method to be called `update_token` [Issue #114](https://github.com/joegasewicz/Flask-JWT-Router/issues/114)
+
```
Comment (kousikmitra) [inline CHANGELOG.md]: remove spaces after list items

**Comment by joegasewicz** [comment]
HI @neilong31 Thanks for the changes, could you just look at @kousikmitra comment and then i can merge your excellent work in, thank you.

**Comment by joegasewicz** [comment]
Thanks @neilong31 for the good work.

**Pull request merged**